mbuild/formats/cassandramcf.py

```diff
@@ -0,0 +1,596 @@
+    structure : parmed.Structure
+        Parmed structure object
+    KCAL_TO_KJ : float
+        Conversion factor from kcal to kJ
+    """
+
+    header = ( '\n!Improper Format\n'
+               '!index i j k l type parameters\n'
+               '!type="harmonic", parms=force_constant equilibrium_improper\n'
+               '\n# Improper_Info\n'
+             )
+
+    mcf_file.write(header)
+    mcf_file.write('{:d}\n'.format(len(structure.impropers)))
+
+    improper_type = 'harmonic'
+    for i, improper in enumerate(structure.impropers):
+        mcf_file.write('{:<4d}  {:<4d}  {:<4d}  {:<4d}  {:<4d}'
+                       '  {:s}  {:8.3f}  {:8.3f}\n'.format(
+            i+1, improper.atom1.idx+1, improper.atom2.idx+1,
+            improper.atom3.idx+1, improper.atom4.idx+1,
+            improper_type, improper.type.psi_k*KCAL_TO_KJ,
+            improper.type.psi_eq))
+
+def _write_fragment_information(mcf_file, structure, frag_list, frag_conn):
+    """Write the fragments in the molecule.
+
+    Parameters
+    ----------
+    mcf_file : file object
+        The file object of the Cassandra mcf being written
+    structure : parmed.Structure
+        Parmed structure object
+    frag_list : list
+        Atom ids belonging to each fragment
+    frag_conn : list
+        Fragment ids of connected fragments
+
+    """
+
+    header = ('\n!Fragment Format\n'
+              '!index number_of_atoms_in_fragment branch_point other_atoms\n'
+              '\n# Fragment_Info\n'
+             )
+
+    mcf_file.write(header)
+
+    # Special cases first
+    if len(frag_list) == 0:
+        if len(structure.atoms) == 1:
+            mcf_file.write('1\n')
+            mcf_file.write('1 1 1\n')
+        elif len(structure.atoms) == 2:
+            mcf_file.write('1\n')
+            mcf_file.write('1 2 1 2\n')
+        else:
+            warnings.warn('More than two atoms present but '
+                          'no fragments identified.')
+            mcf_file.write('0\n')
+    else:
+        mcf_file.write('{:d}\n'.format(len(frag_list)))
+        for i, frag in enumerate(frag_list):
+            mcf_file.write('{:d}    {:d}'.format(i+1, len(frag)))
+            for idx in frag:
+                mcf_file.write('    {:d}'.format(idx+1))
+            mcf_file.write('\n')
+
+    mcf_file.write('\n\n# Fragment_Connectivity\n')
+    mcf_file.write('{:d}\n'.format(len(frag_conn)))
+    for i, conn in enumerate(frag_conn):
+        mcf_file.write('{:d}    {:d}    {:d}\n'.format(i+1, conn[0]+1, conn[1]+1))
+
+def _write_intrascaling_information(mcf_file, lj14, coul14):
+    """Write the intramolecular scaling in the molecule.
+
+    Parameters
+    ----------
+    mcf_file : file object
+        The file object of the Cassandra mcf being written
+    lj14 : float
+        The 1-4 scaling parameter for LJ interactions
+    coul14 : float
+        The 1-4 scaling parameter for Coulombic interactions
+
+    """
+
+    header = ( '\n!Intra Scaling\n'
```

Comment:
Regarding long lines, I think this is a good example of using parentheses to help wrap long lines